ExamplesForHomalg/examples/Coupling.g

```diff
@@ -20,9 +20,9 @@ Dz  \
 M := DiagMat( [ M1, M2, M3 ] );
 M := ShallowCopy( M );	## copy M before setting entries
 SetIsMutableMatrix( M, true );
-SetMatElm( M, 1, 2, "1" );
-SetMatElm( M, 2, 3, "1" );
-SetMatElm( M, 3, 3, "1" );
+M[ 1, 2 ] := "1";
```

Comment:
Circle-ci reports that we need double-semicolons here to suppress the output.

Reply:
That makes sense. Fixed, and also two other similar places. We'll see soon if I overlooked further ones :-)

Reply:
Actually, thinking about it again, I actually do *not* understand: Why does the output need to be suppressed here? How can I test this locally? I think this might also affect these other PRs, but their repositories don't use CI, so I am not sure:
- https://github.com/homalg-project/AbelianSystems/pull/1
- https://github.com/homalg-project/D-Modules/pull/4
- https://github.com/homalg-project/MapleForHomalg/pull/3
- https://github.com/homalg-project/VirtualCAS/pull/2